config/install/user.role.site_admin.yml

```diff
@@ -0,0 +1,7 @@
+langcode: en
+status: true
+dependencies: {  }
+id: site_administrator
```

Comment:
will the mismatch between calling the user.role site_admin and the id site_administrator cause any issues - should we stadnardise - either the file name or the id?

Reply:
I agree, makes sense to keep the role and id matching as it does for the other roles in your request.